packages/jaeger-ui/src/components/QualityMetrics/ScoreCard.test.js

```diff
@@ -13,9 +13,8 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow } from 'enzyme';
-
 import ScoreCard from './ScoreCard';
+import ReactShallowRenderer from '../../utils/helpers/ReactShallowRenderer';
```

Comment:
why not define a replacement `shallow` function there?

Reply:
Done. Boiled the code to even simpler usage.
Previously, I was using the code as described in this [guide](https://thesametech.com/snapshot-testing-in-rtl/#:~:text=Snapshot%20testing%20is%20often%20used,operate%20with%20the%20real%20DOM.).

Now, only the shallow function's import statement needs to be changed.